src/views/Search/Record.vue

```diff
@@ -1,18 +1,36 @@
 <template>
-    <div class="standard" id="standard_ABA">
-        <h2>ABA Adult Mouse Brain</h2>
+    <div class="standard" :id="content['fairsharingRecord'].id" v-if="content">
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['v-if' should be on a new line.](https://app.codacy.com/gh/FAIRsharing/fairsharing.github.io/pullRequest?prid=4893952)